src/com/gfredericks/test/chuck/clojure_test.cljc

```diff
@@ -0,0 +1,104 @@
+(ns com.gfredericks.test.chuck.clojure-test
+  (:require [clojure.test.check :as tc]
+            [clojure.test.check.properties :as prop
+             #?@(:cljs [:include-macros true])]
+            #?(:clj  [clojure.test :refer [is]]
+               :cljs [cljs.test :refer-macros [is]])))
+
+;; copied from clojure.test.check, which privatized the function in
+;; recent versions.
+;;
+;; I think there might be plans for test.check to abstract this logic
+;; into a protocol or something, so I'm not too bothered by the
+;; copypasta for now.
```

Comment:
You are right, it was removed on master. It slipped through here... I just removed it in 8f978c6